scripts/build_shapes.py

```diff
@@ -1379,4 +1429,10 @@ def gadm(
         nprocesses=nprocesses,
         simplify_gadm=simplify_gadm,
     )
+
+    if snakemake.params.get("subregion").get("define_by_gadm", False):
```

Comment:
```suggestion
    if snakemake.params["subregion"]["define_by_gadm"]:
```
Could we reformulate like this?